backend/canisters/user_index/impl/src/queries/current_user.rs

```diff
@@ -43,6 +44,9 @@ fn current_user_impl(state: &RuntimeState) -> Response {
             diamond_membership_details: u.diamond_membership_details.hydrate(now),
             diamond_membership_status: u.diamond_membership_details.status_full(now),
             moderation_flags_enabled: u.moderation_flags_enabled,
+            chit_balance: u.chit_balance,
+            streak: u.streak.days(now),
+            next_daily_claim: if u.streak.can_claim(now) { midnight_today(now) } else { midnight_tomorrow(now) },
```

Comment:
Surely if they can claim now then the next daily claim is yesterday midnight? Or simply just now?

Reply:
renamed